### PR TITLE
Complete build for download

### PR DESCRIPTION
**Pull request by tomheller**
* Adds `package` gulp task that creates zip file of `main.js` and `main.css`
* Adds download page with usage options
* Adds conversion tracking GA event when downloading groundhog.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -221,4 +225,12 @@ gulp.task('icons', function() {
     .pipe(gulp.dest('dist/assets/images/icons'))
     .pipe(sprites({ templates: ['default-svg']}))
     .pipe(gulp.dest('dist/assets/images'));
-})
+});
+
```
Comment (ddprrt) [inline gulpfile.js]: REMOVE DA SPACE!
giphy

File: gulpfile.js [hunk 2]
```diff
@@ -221,4 +225,12 @@ gulp.task('icons', function() {
...
+
+gulp.task('package', () => {
+  const version = packageData.version;
+  gulp.src(['./dist/js/main.js', './dist/css/main.css'])
```
Comment (ddprrt) [inline gulpfile.js]: Wonder if we should version that stuff (js and css) as well...

File: docs/_pages/download.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+And include the JS at the end of the `<body>` tag of your page:
+
+```html
+<body>
+... all your content
+
+<script type="text/javascript" src="main.js"></script>
+</body>
+```
+
```
Comment (ddprrt) [inline docs/_pages/download.md]: Comment about getting da assets?

**Comment by ddprrt** [comment]
Merge ma?

**Comment by ddprrt** [comment]
@tomheller Wink amoi!

**Comment by tomheller** [comment]
@ddprrt öhm, i hab das voll übersehen gestern. i glaub wir können mergen.

**Pull request merged**